tests/helpers/module-for-acceptance.js

```diff
@@ -2,22 +2,45 @@ import { module } from 'qunit';
 import Ember from 'ember';
 import startApp from '../helpers/start-app';
 import destroyApp from '../helpers/destroy-app';
+import FactoryGuy, { manualSetup } from 'ember-data-factory-guy';
+import config from 'ember-get-config';
+import FakeServer, { stubRequest } from 'ember-cli-fake-server';
 
 const { RSVP: { Promise } } = Ember;
 
 export default function(name, options = {}) {
   module(name, {
     beforeEach() {
-      this.application = startApp();
-
-      if (options.beforeEach) {
-        return options.beforeEach.apply(this, arguments);
-      }
+        this.application = startApp();
+        FakeServer.start();
+        manualSetup(this.application.__container__);
+        const url = config.OSF.apiUrl;
+        const provider = FactoryGuy.build('preprint-provider');
+        stubRequest('get', url + '/v2/users/me', (request) => {
```

Comment:
This looks helpful!

I ran the tests in chrome with the network inspector open, and noticed that the page still sends a request to google analytics every time tests run. This may indicate one more thing that needs stubbing. Thoughts?